elasticache.tf

```diff
@@ -7,6 +7,10 @@ resource "aws_elasticache_cluster" "novelcovid_redis" {
   parameter_group_name = "default.redis3.2"
   engine_version       = "3.2.10"
   port                 = 6379
+  subnet_group_name    = aws_subnet.master[0].id
```

Comment:
Can I ask some questions to catch up?

Looking at this it seems the static content is killing us and we are planning on implementing caching on our ingress that is backed by redis.  Is that correct?

I haven't reviewed this completely so may be wrong here, but this won't work will it?  The master subnet you are referencing is defined in a module way down deep in the EKS stuff.  I don't think this will have any value here.

Reply:
I think if you want to use this id here you need to:

1. Make this resource dependent upon that eks module resource (not sure that's supported but maybe) and
2. Export these ids as output from that module and
3. Reference that output, not the value here.

Of course, terraform has made advancements in this area but as defined, there is no `aws_subnet.master` at all in the scope of this resource.

Reply:
Sure. So this PR doesn't have to do with static asset caching. Yesterday we pointed gatling at the cluster and had it request static resources and make api calls. Gatling was configured to send requests at the same rate we were seeing during the cutover yesterday morning.  Our edge/ingress actually handled this ok after scaling them up, but the novelcovid.redis instance fell over with OOM errors. We tried allocating more memory but could not get it to stay up. Figured standing up a managed redis instance for novelcovid was a quicker/easier solution than fiddling with resources.

Reply:
As for this PR, I'm not sure :) When I tried to point the novelcovid backend at this redis instance I was seeing timeouts, and thought maybe it couldn't connect because the subnet matters here.  
![image](https://user-images.githubusercontent.com/22768918/85022326-2631a600-b141-11ea-8dd7-05c7de020bd9.png)

Reply:
Here is the last run from yesterday's experiment. Static resources get returned successfully 99% of the time.

![_Users_hiromi_results_basicsimulation-20200617213215166_index html](https://user-images.githubusercontent.com/5401333/85023116-ec10d600-b139-11ea-8b93-0dd6131ab779.png)

Reply:
Okay, understood on the purpose of this now. 

I also think it may be a network issue.  However, this probably is insufficient to fix that as I don't think it will even run, will it?

Reply:
I haven't looked at any of this code yet, but did you guys build security groups to allow the communication between the EKS nodes and Elasticache?